Database/Corrections/sodQuestFixes.lua

```diff
@@ -3164,10 +3164,70 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.questLevel] = 15,
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.SHAMAN,
-            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receieve the rune."},
+            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receive the rune."},
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90209] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{414624}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Lighthouse Stash under the right of the tree to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90210] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{407731}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Stonemason's Toolbox on the platform behind the quest givers at Stonewrough Dam to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90211] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {{210549}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
```

Comment:
[Pick Pocket](https://www.wowhead.com/classic/spell=921/pick-pocket) is learned at level 4.

```suggestion
            [questKeys.requiredLevel] = 4
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5609/commits/82f0883c19e082887899e9a1192485228c484edf